### ISSUE TITLE
[Feature] Add the Bildungscampus Mensa in Campus Heilbronn

### ISSUE THREAD

**Issue by PermissionError** [TUM-Dev/eat-api#220]
**Is your feature request related to a problem? Please describe.**
The App currently does not include information about the Bildungscampus Mensa at Campus Heilbronn.

**Describe the solution you'd like**
The Studierendenwerk Heildelberg runs the Bildungscampus Mensa, and the opening times and daily menus are available on their website. It should be feasible to pull this from the website or their API directly if there is one, and add it in a similar fashion to the other Mensas. Additionally, the app should be able to use the existing location feature to automatically display the Heilbronn Mensa if the user is in Heilbronn.

**Describe alternatives you've considered**
There is a separate "Mein Bildungscampus" app that provides public transport timetables and parking information for Campus Heilbronn. However, the Mensa is not covered and that App is not open source, therefore contribution is not possible.

**Additional context**
I am happy to contribute this feature myself, but may take some time as I learn the structure of the app source code.

**Comment by CommanderStorm** [comment]
@tobiasjungmann no need to open another issue if admins can just transfer the issues ^^

Note that currently, the opening times are manually collected.
If you give me the correct opening times, I can add them.
See https://github.com/TUM-Dev/Campus-Backend/issues/82 for further details

**Comment by PermissionError** [comment]
> @tobiasjungmann no need to open another issue if admins can just transfer the issues ^^
> 
> Note that currently, the opening times are manually collected (not an idal system, can agree).
> If you give me the correct opening times, I can add them.
> See https://github.com/TUM-Dev/Campus-Backend/issues/82 for further details

Sure, the opening times of the Bildungscampus Mensa is weekdays 11:00 to 14:30. I plan on making its ID in eat-api "mensa-bildungscampus-heilbronn", if that is of any relevance.

**Comment by tobiasjungmann** [comment]
Transferring issues is a nice option, I did not know this existed. That's good to know for the next issue! @PermissionError That looks like a fitting API-Key.

**Comment by PermissionError** [comment]
Actually it seems like the OpenMensa parser set for StuWerk Heidelberg already covers the Bildungscampus Mensa. Could eat-api just access the OpenMensa API directly?

**Comment by PermissionError** [comment]
> The meal plans are currently queried by the [eat-api](https://github.com/TUM-Dev/eat-api). Every canteen provided in this api will automatically be included in this app. It would be great if you could add it there! Please open the same issue there. The corresponding codebase is a bit smaller too :D

Hi, I noticed that even after the automated deployment today the Campus App did not update. Upon further research I found an open issue and a corresponding open draft PR for "migrating to eat-api" in TUM-Dev/Campus-Android. Are the apps already using eat-api? The API URL I found in Campus-Android seems to go to https://www.devapp.it.tum.de/mensaapp/, which I cannot find the source code of.

**Comment by COM8** [comment]
@PermissionError the Android app right now does not support the eat-api and it probably never will. But the iOS and Windows apps do.
The reason for this is that we are currently working on a replacement for it based on Flutter ([TUM Campus App with Flutter](https://github.com/TUM-Dev/Campus-Flutter)) which is almost ready to be shipped.

Regarding English translations. Since we only have a limited quota for deepl, the English translations CI runs only once per week. I manually triggered it so translations should be available shortly.

**Comment by CommanderStorm** [comment]
See https://github.com/TUM-Dev/Campus-Android?tab=readme-ov-file#-public-service-announcement- why we are migrating to a new app
